__tests__/app.test.js

```diff
@@ -199,4 +199,71 @@ describe('POST /api/articles/:article_id/comments', () => {
             expect(body.message).toBe("Not found");
         });
     });
+});
+describe('PATCH /api/articles/:article_id', () => {
+    it('returns status 200 and updated article with new votes value', () => {
+        const returnedArticle = {
+            article_id: 1,
+            title: 'Living in the shadow of a great man',
+            topic: 'mitch',
+            author: 'butter_bridge',
+            body: 'I find this existence challenging',
+            created_at: "2020-07-09T20:11:00.000Z",            
+            votes: 101,
+            article_img_url:
+              'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+        }
+        return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then(({body}) => {
+            console.log(body);
+            expect(body[0]).toEqual(returnedArticle);
```

Comment:
You shouldnt need to assert exactly what this will be like here, you can instead assert how it will look i.e. what data type each element is using expect.objectContaining or .toMatchObject